Makefile

```diff
@@ -31,10 +31,10 @@ npm_install: package.json
 frontend_build:
 	npm run --silent frontend-build:${GULP_ENVIRONMENT}
 
-test: show_environment test_pep8 test_python test_javascript
+test: show_environment test_flake8 test_python test_javascript
 
-test_pep8: virtualenv
```

Comment:
So PyFlakes actually runs a pep8 check as well as a lint check. I can't see anyone needing to use pep8 as well as the flake8 command. Happy to add it back in later if people realise they want it but hopefully devs will barely notice the change :crossed_fingers: .